Bottom: 76ab701b584353b1ebc07f8bc9cfff4e0e43bb0c
Top:    12802cbdc0deee0eb2183e0d010f142077fbaff0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-28 23:51:57 +0100

Refresh of hid-ps-don-t-use-the-core-s-dr

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index dfeb58f..3b7477f 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -664,8 +664,7 @@ ps_expose (void)
         continue;
 
       if (set_layer (0, group))
-        if (DrawLayerGroup (group, NULL))
-          DrawPPV (group, NULL);
+        DrawLayerGroup (group, NULL);
     }
 
   CountHoles (&nplated, &nunplated, NULL);
